### PR TITLE
Use raw strings in EditorFeatures tests

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**

### CODE REVIEW

File: src/EditorFeatures/CSharpTest/AddMissingImports/CSharpAddMissingImportsRefactoringProviderTests.cs [hunk 1]
```diff
@@ -63,281 +63,281 @@ protected override void InitializeWorkspace(TestWorkspace workspace, TestParamet
         [WpfFact]
```
Comment (CyrusNajmabadi) [inline src/EditorFeatures/CSharpTest/AddMissingImports/CSharpAddMissingImportsRefactoringProviderTests.cs:63]: view with whitespace off.

**Review by tmat — CHANGES_REQUESTED** [review]
Could you please revert changes in EditAndContinue tests?

This would produce a lot of conflicts with my work in progress. We can apply the refactoring later.

**Comment by CyrusNajmabadi** [comment]
Yup. Happy to.

**Comment by CyrusNajmabadi** [comment]
@tmat enc changes reverted.

**Review by tmat — APPROVED** [review]

**Pull request merged**